CHANGELOG.md

```diff
@@ -51,6 +51,7 @@ IMPROVEMENTS:
   * provider/digitalocean: Make user_data force a new droplet [GH-3740]
   * provider/vsphere: Do not add network interfaces by default [GH-3652]
   * provider/openstack: Configure Fixed IPs through ports [GH-3772]
+  * provider/openstack: Specify a port ID on a Router Interface [GH-3903]
```

Comment:
I don't spot any changes to the openstack provider in this pull request - is this a remnant of another branch? In general we'd prefer it if pull requests didn't update the CHANGELOG, as it tends to lead to frequent merge conflicts. Could you remove this from the PR?